packages/synapse-react-client/src/components/DownloadCart/DownloadDetails.tsx

```diff
@@ -17,13 +18,22 @@ export default function DownloadDetails(props: DownloadDetailsProps) {
   const iconClassName = isInactive ? 'SRC-inactive' : 'SRC-primary-text-color'
   return (
     <span className="DownloadDetailsV2">
-      <span className="item">{!isInactive && <> {numFiles} Files </>}</span>
+      <span className="item">
+        {!isInactive && (
+          <>
+            {numFiles.toLocaleString()} {pluralize('File', numFiles)}
+          </>
+        )}
```

Comment:
Use `toLocaleString` on numbers that will be displayed to the user (e.g. `1000000` would render as "1,000,000" instead of "1000000").

Use pluralize to handle the case where there is only one file (returns "file" instead of "files").

Reply:
Thanks Nick for the explanation. I also noticed that the number format is something like 1_000_000, instead of 1000000 which is interesting.